readme.md

```diff
@@ -1,5 +1,113 @@
+
+The mission of the .NET Foundation is to foster a community of .NET developers focused on collaborating on commercially friendly open source software projects that enhance the .NET platform and create value within the .NET ecosystem.
+
+## Rationale
+
+An important aspect of the .NET Foundation's vision is to attract and foster vibrant communities of open source projects and contributors. However, in order to preserve the relevancy and credibility of the .NET Foundation's brand and reputation, it must ensure that all member projects fully embrace its mission and values and are willing to function as first class ambassadors and representatives of the .NET Foundation. It is the role of the Project Committee to assist the Board of Directors in ensuring this alignment.
+
+## Responsibilities
+
+The Project Committee manages the day to day requirements of Projects. This includes but is not limited to the following:
+
+- Review and evaluate new Project applications
+- Monitor health of existing Projects
+- Handle the retirement of Projects
+- Provide support to Projects
+
+## Organizational Structure
+
+The Project Committee is comprised of volunteer .NET Foundation members and has a designated Chairperson appointed by the Executive Director. The Chairperson ensures that the Project Committee meets monthly to discuss and review items pertaining to Projects and provides information and recommendations to the Executive Director so that they can be presented to the Board of Directors for approval. All official communication from the .NET Foundation pertaining to Projects must be delivered by the Executive Director.
+
+## Evaluation Criteria
+
+Transparency is a key open source characteristic, and as a result the criteria for evaluating .NET Foundation projects is designed to be as measurable and objective as possible. The criteria is not only important for evaluating new project applications but for measuring the ongoing health of existing projects. There are two types of criteria which have been defined:
+
+**Eligibility** - this criterion is binary in nature and is used to determine if a project is fully aligned with the mission and values of the .NET Foundation.
+
+**Health** - this criterion is quantitative in nature and is based on publicly accessible metrics which can be used to evaluate the health and maturity of a project.
+
+## Project Levels
+
+Projects are categorized according to the following levels: 
+
+**Applicant** - a project which has applied to the .NET Foundation to become a member. The Project Committee will review the application within one month of the submission to determine whether or not the project has satisfied the Eligibility and/or Health requirements. Any feedback to the applicant will be delivered by the Executive Director.
```

Comment:
The Project Committee should be entitled to comment on the application issue, ask questions, seek clarification, etc. Comm's don't need to go though the ED.